assets/sass/widgets/_googlesitekit-widget-analyticsAllTraffic.scss

```diff
@@ -201,6 +201,19 @@
 		}
 	}
 
+	.googlesitekit-widget--analyticsAllTraffic__dimensions-chart-gathering-data {
+		color: $c-silver-2;
+		font-size: 18px;
+		font-weight: 400;
```

Comment:
@hussain-t This is redundant since the `font-weight` is already set to `400` by default.

Reply:
Done ✅